bindings/node.js/kzg.cxx

```diff
@@ -124,46 +151,70 @@ Napi::Value FreeTrustedSetup(const Napi::CallbackInfo& info) {
   return env.Undefined();
 }
 
-// blobToKzgCommitment: (blob: Blob, setupHandle: SetupHandle) => KZGCommitment;
+/**
+ * Convert a blob to a KZG commitment.
+ *
+ * @param {Blob} blob - The blob representing the polynomial to be committed to
+ *
+ * @returns {KZGCommitment} -  The resulting commitment
+ *
+ * @throws {TypeError} - for invalid arguments or failure of the native library
+ */
 Napi::Value BlobToKzgCommitment(const Napi::CallbackInfo& info) {
-  auto env = info.Env();
-
+  Napi::Env env = info.Env();
   size_t argument_count = info.Length();
   size_t expected_argument_count = 2;
   if (argument_count != expected_argument_count) {
     return throw_invalid_arguments_count(expected_argument_count, argument_count, env);
   }
-
-  Blob *blob = (Blob *)extract_byte_array_from_param(info, 0, "blob");
-  if (env.IsExceptionPending()) {
+  Blob *blob = get_blob(env, info[0]);
+  if (blob == nullptr) {
     return env.Null();
   }
-
+  if (!info[1].IsExternal()) {
+     Napi::TypeError::New(env, "Must pass setupHandle as the last function argument").ThrowAsJavaScriptException();
+  }
   auto kzg_settings = info[1].As<Napi::External<KZGSettings>>().Data();
 
   KZGCommitment commitment;
   C_KZG_RET ret = blob_to_kzg_commitment(&commitment, blob, kzg_settings);
   if (ret != C_KZG_OK) {
      Napi::Error::New(env, "Failed to convert blob to commitment")
       .ThrowAsJavaScriptException();
-    return env.Undefined();
+    return env.Null();
   };
 
-  return napi_typed_array_from_bytes((uint8_t *)(&commitment), BYTES_PER_COMMITMENT, env);
+  return Napi::Buffer<uint8_t>::Copy(env, reinterpret_cast<uint8_t *>(&commitment), BYTES_PER_COMMITMENT);
```

Comment:
Fixed the conflicts after merging in main. But some of the tests fail now because the return type has been changed from `Uint8Array` to `Buffer`. It's pretty simple to fix if you modify the tests, but `KZGCommitment` & `KZGProof` are defined as `Uint8Array` at the top of the `kzg.ts` file, so that's what I would expect.

Reply:
Thanks for merging that in @jtraglia.  Its updated, and all tests are passing again.